tutor/specs/components/template-modal.spec.js

```diff
@@ -0,0 +1,9 @@
+import TemplateModal from '../../src/components/template-modal';
+
+describe('Template Modal', () => {
+
+  it('renders and matches snapshot', () => {
+    expect.snapshot(<TemplateModal templateType="neutral" />).toMatchSnapshot();
+  });
+
+});
```

Comment:
this isn't actually rendering anything, see the `template-modal.spec.js.snap` file above.  It must be returning null

Reply:
Whooooops, thanks!